src/main/java/gov/usgs/locator/Event.java

```diff
@@ -186,6 +187,11 @@ public boolean readHydra(String eventID) {
 		Pick pick;
 		Pattern affinity = Pattern.compile("\\d*\\.\\d*");
 		
+		// setup event path
+		if (eventPath != null) {
+			LocUtil.eventPath = eventPath;
```

Comment:
see other related changes

Reply:
This one is correct, if the provided event path does not equal null, then set the LocUtil.eventPath to it, otherwise let the default value in LocUtil.eventPath stand (if the provided event path is null)